RELEASE_NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Release Notes
 
+## 0.0.11
+
+* fix: indefinite retrials while shutdown of client.
```

Comment:
It was not indefinite since the queues are finite. Now it's more like - "duration of client shutdown is configurable".